gdal/frmts/tiledb/tiledbdataset.cpp

```diff
@@ -1264,16 +1264,28 @@ GDALDataset *TileDBDataset::Open( GDALOpenInfo * poOpenInfo )
 
         char ** papszStructMeta = poDS->GetMetadata( "IMAGE_STRUCTURE" );
         const char* pszXSize = CSLFetchNameValue( papszStructMeta, "X_SIZE");
-        if ( pszXSize )
+        if ( pszXSize > 0 )
```

Comment:
test should be on RasterXSize. Same for y

Reply:
yes, thank you, my mistake.